spec/src/main/doc/core/implementation.asciidoc

```diff
@@ -730,7 +494,7 @@ A bean constructor may have any number of parameters. All parameters of a bean c
 
 === Injected fields
 
-An _injected field_ is a non-static, non-final field of a bean class or of any Java EE component class supporting injection.
+An _injected field_ is a non-static, non-final field of a bean class.
```

Comment:
You're right, they should be removed. My only concern is for `InjectionTarget`s created at runtime with `BeanManager#CreateInjectionTarget` or indirectly via `Unmanaged`. Some of this rules may apply to the classes used to create these InjectionTarget. wdyt ?